hubs/widgets/fxos.py

```diff
@@ -0,0 +1,39 @@
+from hubs.widgets.chrome import panel
+from hubs.widgets.base import argument
+
+import jinja2
+
+import hubs.validators as validators
+
+
+template = jinja2.Template("""
+    <script>
+    $(document).ready(function (){
+        $('#mozinstall').click(function() {
+            navigator.mozApps.install("http://127.0.0.1:8000/static/manifest.webapp");
```

Comment:
And, it's hardcoded here too?  We can't have the code be duplicated in three places, I think.